arch/risc-v/src/esp32c3/esp32c3_ble_adapter.c

```diff
@@ -0,0 +1,2059 @@
+ *   Create message queue
+ *
+ * Input Parameters:
+ *   queue_len - queue message number
+ *   item_size - message size
+ *
+ * Returned Value:
+ *   Message queue data pointer
+ *
+ ****************************************************************************/
+
+void *queue_create_wrapper(uint32_t queue_len, uint32_t item_size)
+{
+  struct mq_attr attr;
+  struct mq_adpt *mq_adpt;
+  int ret;
+
+  mq_adpt = kmm_malloc(sizeof(struct mq_adpt));
+  if (!mq_adpt)
+    {
+      wlerr("ERROR: Failed to malloc\n");
+      return NULL;
+    }
+
+  snprintf(mq_adpt->name, sizeof(mq_adpt->name), "/tmp/%p", mq_adpt);
+
+  attr.mq_maxmsg  = queue_len;
+  attr.mq_msgsize = item_size;
+  attr.mq_curmsgs = 0;
+  attr.mq_flags   = 0;
+
+  ret = file_mq_open(&mq_adpt->mq, mq_adpt->name,
+                          O_RDWR | O_CREAT, 0644, &attr);
+
+  if (ret < 0)
+    {
+      wlerr("ERROR: Failed to create mqueue\n");
+      kmm_free(mq_adpt);
+      return NULL;
+    }
+
+  mq_adpt->msgsize = item_size;
+  return (void *)mq_adpt;
+}
+
+/****************************************************************************
+ * Name: queue_send_wrapper
+ *
+ * Description:
+ *   Generic send message to queue within a certain period of time
+ *
+ * Input Parameters:
+ *   queue - Message queue data pointer
+ *   item  - Message data pointerint
+ *   ticks - Wait ticks
+ *   prio  - Message priority
+ *
+ * Returned Value:uint32_t
+ *   True if success or false if fail
+ *
+ ****************************************************************************/
+
+int queue_send_wrapper(void *queue, void *item, uint32_t block_time_ms)
+{
+  return esp_queue_send_generic(queue, item, block_time_ms, 0);
+}
+
+/****************************************************************************
+ * Name: queue_delete_wrapper
+ *
+ * Description:
+ *   Delete message queue
+ *
+ * Input Parameters:
+ *   queue - Message queue data pointer
+ *
+ * Returned Value:
+ *   None
+ *
+ ****************************************************************************/
+
+void queue_delete_wrapper(void *queue)
+{
+  struct mq_adpt *mq_adpt = (struct mq_adpt *)queue;
+
+  file_mq_close(&mq_adpt->mq);
+  file_mq_unlink(mq_adpt->name);
+  kmm_free(mq_adpt);
+}
+
+/****************************************************************************
+ * Name: esp_bt_controller_init
+ *
+ * Description:
+ *    Init  BT controller.
+ * Input Parameters:
+ *    cfg -  Initial configuration of BT controller.
+ *
+ * Returned Value:
+ *    None
+ *
+ ****************************************************************************/
+
+int esp_bt_controller_init(esp_bt_controller_config_t *cfg)
+{
+  int err = -1;
+
+  if (btdm_controller_status != ESP_BT_CONTROLLER_STATUS_IDLE)
+    {
+      return -1;
+    }
+
+  if (cfg == NULL)
+    {
+      return -1;
+    }
+
+  if (cfg->controller_task_prio != 110
+        || cfg->controller_task_stack_size < 4096)
+    {
+      wlerr("ERROR:Invalid controller task prioriy or stack size");
```

Comment:
```suggestion
      wlerr("Invalid controller task prioriy or stack size");
```
Do not add `ERROR` and similar prefixes to these functions